cmd/mesh/main.go

```diff
@@ -21,10 +21,10 @@ import (
 type standaloneConfig struct {
 	// WSRPCAddr is the interface and port to use for the JSON-RPC API over
 	// WebSockets. By default, 0x Mesh will listen on localhost and port 60557.
-	WSRPCAddr string `envvar:"WS_RPC_ADDR" default:"localhost:60557"`
+	WSRPCAddr string `envvar:"WS_RPC_ADDR" default:"ws://localhost:60557"`
 	// HTTPRPCAddr is the interface and port to use for the JSON-RPC API over
 	// HTTP. By default, 0x Mesh will listen on localhost and port 60556.
-	HTTPRPCAddr string `envvar:"HTTP_RPC_ADDR" default:"localhost:60556"`
+	HTTPRPCAddr string `envvar:"HTTP_RPC_ADDR" default:"http://localhost:60556"`
```

Comment:
Why is this change strictly necessary? I'd be against it if it's optional since it introduces a breaking change to the config envs of Mesh that _many_ people rely on.